### DISCUSSION TITLE
How to deploy it on Netlify

### DISCUSSION THREAD

**Discussion by jli113** [Ice-Hazymoon/hugo-theme-luna#63] [Q&A] [answered]
I want to deploy it on Netlify, but keep getting errors when building the site.

**Comment by Ice-Hazymoon** [comment]
Please describe your problem and error message in detail

**Comment by Ice-Hazymoon** [comment, marked as answer]
Now supported.

https://github.com/Ice-Hazymoon/hugo-theme-luna/blob/master/exampleSite/netlify.toml
